code/datums/antagonists/datum_traitor.dm

```diff
@@ -41,12 +41,22 @@
 			if(!silent)
 				to_chat(traitor_mob, "Your training has allowed you to overcome your clownish nature, allowing you to wield weapons without harming yourself.")
 			traitor_mob.dna.remove_mutation(CLOWNMUT)
+	if(owner.assigned_role == "Mime")
+		var/mob/living/carbon/human/traitor_mob = owner.current
+		if(traitor_mob && istype(traitor_mob))
+			if(!silent)
+				to_chat(traitor_mob, "Your vow of silence is not more important than your goals, therefore it has been lifted.")
```

Comment:
You can check if they are traitors by making them scream. Please make it toggleable

Reply:
A true mime refuses the scream.

I can make traitor mimes fake not screaming.